live-build/misc/ansible-roles/appliance-build.challenge-response-authentication/tasks/main.yml

```diff
@@ -0,0 +1,18 @@
+#
+# Copyright 2019 Delphix
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+---
+- command: /opt/delphix/server/bin/register/cra_control enable
```

Comment:
Why do we want this to be in a new `challenge-response-authentication` role, rather than included in the existing `virtualization-external` role?

Reply:
Also, would it be better to just run:
```
pam-auth-update --enable challenge-response
```
rather than using this script?

I worry that change to that script, and the assumption it makes about its run time environment (e.g. it expecting to only run on a running DE, rather than this chroot/build environment), might result in build failures later (e.g. even if works now, maybe it'll break later).

Reply:
I didn't notice that existing role :) Will consolidate.

I can run `pam-auth-update` directly. Will do in the next diff.